### ISSUE TITLE
Can not Remove Ultimate Item Filter

### ISSUE THREAD

**Issue by P3YC1** [xenondevs/Nova#371]
### Agreements

- [X] I am using an official build of Nova.
- [X] I am running the latest version of Nova.
- [X] I am using a [server software supported by Nova](https://docs.xen.cx/nova/admin/compatibility/#compatible-server-software)
- [X] I am running the latest version of my server software.
- [X] I have checked for similar issues.

### Check for plugin incompatibilites

There are no other plugins running on my server.

### Plugin Version, Server Software, Server Version

Nova: 0.16 RC 2, Paper: git-Paper-463 (MC: 1.20.4)

### Nova Addons and Versions

Jetpacks v0.1.12-RC.2, Logistics v0.2.8-RC.1, Machines v0.4.5-RC.4, Simple-Upgrades v1.3-RC.2, Vanilla Hammers v1.4-RC.2

### Expected behavior

It is possible to take an Ultimate Item Filter out of a Basic Cable that is Attached to a Chest, while the Ultimate Item Filter has a damaged Crossbow inside it.

### Observed/Actual behavior

It is **NOT** possible to take an Ultimate Item Filter out of a Basic Cable that is Attached to a Chest, while the Ultimate Item Filter has a damaged Crossbow inside it. It is stuck inside there as it would belong to the UI.

### Steps to reproduce

1. Get a Crossbow
2. Shoot with it some time until it is damaged
3. Get an Ultimate Item Filter, a Chest, and a Basic cable.
4. Place the Crossbow inside the ultimate item filter.
5. Place the chest, connect the cable to it, and place the ultimate item filter inside
6. Try to take out the ultimate item filter

### Server log

[latest.log](https://github.com/xenondevs/Nova/files/15033794/latest.log)

**Comment by P3YC1** [comment]
Okay, so I did some more testing. It seems the issue occurs when you have the Item Filter in hand while trying to open the Config for the Cable. If you then look at the cable from a certain position, you can see the filter in your hand twitching, as you open the config for the Cable. If that happens, and you put the cable into the config, you can not retrieve it, aside from breaking the chest. It probably has something to do with the plugin trying to open two menus at the same time.
If you can't reproduce it, I'll try to get a video of the bug happening.

**Comment by NichtStudioCode** [comment]
Yeah, I still can't reproduce it. It'd be great if you could send a video.

**Comment by P3YC1** [comment]
https://github.com/xenondevs/Nova/assets/58711968/ff7f051c-96ec-4a4c-9fa6-f034b2baeabc
